src/containers/Statistics/StatisticsLinePlot.tsx

```diff
@@ -3,34 +3,105 @@ import LinePlotContainer from '@components/LinePlotContainer/LinePlotContainer'
 import { useDispatch, useSelector } from 'react-redux'
 import { action } from '@reducers/stats'
 import statsSelector from '@selectors/stats'
-import { getCollateralValue } from '@selectors/exchange'
 
 export const StatisticsLinePlot: React.FC = () => {
   const dispatch = useDispatch()
   const statsData = useSelector(statsSelector.linePlot)
-  const collValLastMinDay = useSelector(statsSelector.tvl)
-  const tvlValue = useSelector(getCollateralValue)
-  const [tvlData, setTvlData] = React.useState<{ value: number; percent: string }>({
+  const [timeActive, setTimeActive] = React.useState<{
+    index: number
+    serieId: string
+    timestamp: number
+    value: number
+  }>({
+    index: 0,
+    serieId: 'dafault',
+    timestamp: 0,
+    value: 0
+  })
+  const [menuOption, setMenuOption] = React.useState('Volume')
+  const [infoData, setInfoData] = React.useState<{ name: string; value: number; percent: string }>({
+    name: menuOption,
     value: 0,
     percent: '0'
   })
   React.useEffect(() => {
     dispatch(action.updateData())
   }, [])
 
+  const findEarlierRekord = () => {
+    return statsData.find(element => element.id === timeActive.serieId)?.points[
+      timeActive.index - 1
```

Comment:
what if "timeActive.index" is equal to 0?

Reply:
return undefined and I wrote a condition for it